common/djangoapps/course_modes/models.py

```diff
@@ -123,7 +136,7 @@ class Meta(object):
     # "honor" to "audit", we still need to have the shoppingcart
     # use "honor"
     DEFAULT_SHOPPINGCART_MODE_SLUG = HONOR
-    DEFAULT_SHOPPINGCART_MODE = Mode(HONOR, _('Honor'), 0, '', 'usd', None, None, None)
+    DEFAULT_SHOPPINGCART_MODE = Mode(HONOR, _('Honor'), 0, '', 'usd', None, None, None, None)
```

Comment:
The Mode() is a namedtuple, so I think we have to include the new field wherever a new Mode is being created?  I was running into some test failures on my local machine so I went ahead and updated all the namedtuples I could find.  Might be overkill, though?